python/ray/serve/_private/constants.py

```diff
@@ -16,19 +16,21 @@
 SERVE_NAMESPACE = "serve"
 
 #: HTTP Address
-DEFAULT_HTTP_ADDRESS = "http://127.0.0.1:8000"
+DEFAULT_HTTP_ADDRESS = os.environ.get(
+    "RAY_SERVE_DEFAULT_HTTP_ADDRESS", "http://127.0.0.1:8000"
+)
 
 #: HTTP Host
-DEFAULT_HTTP_HOST = "127.0.0.1"
+DEFAULT_HTTP_HOST = os.environ.get("RAY_SERVE_DEFAULT_HTTP_HOST", "127.0.0.1")
 
 #: HTTP Port
-DEFAULT_HTTP_PORT = 8000
+DEFAULT_HTTP_PORT = int(os.environ.get("RAY_SERVE_DEFAULT_HTTP_PORT", 8000))
 
 #: Uvicorn timeout_keep_alive Config
 DEFAULT_UVICORN_KEEP_ALIVE_TIMEOUT_S = 5
```

Comment:
there are more here and below, happy to expose all of them if it makes sense to do so

Reply:
we can leave the surface area minimal for now